webapp/components/Paginate/Paginate.spec.js

```diff
@@ -1,39 +1,35 @@
 import { mount } from '@vue/test-utils'
 
-import Paginate from './Paginate'
+import PaginationButtons from './PaginationButtons'
 
 const localVue = global.localVue
 
-describe('Paginate.vue', () => {
-  let propsData, wrapper, nextButton, backButton
-
-  beforeEach(() => {
-    propsData = {}
-  })
+describe('PaginationButtons.vue', () => {
+  let propsData = {}
+  let wrapper
+  let nextButton
+  let backButton
 
   const Wrapper = () => {
-    return mount(Paginate, { propsData, localVue })
+    return mount(PaginationButtons, { propsData, localVue })
   }
-  describe('mount', () => {
-    beforeEach(() => {
-      wrapper = Wrapper()
-    })
 
+  describe('mount', () => {
     describe('next button', () => {
       beforeEach(() => {
         propsData.hasNext = true
         wrapper = Wrapper()
-        nextButton = wrapper.findAll('.ds-button').at(0)
+        nextButton = wrapper.find('[data-test="next-button"]')
       })
 
       it('is disabled by default', () => {
         propsData = {}
         wrapper = Wrapper()
-        nextButton = wrapper.findAll('.ds-button').at(0)
+        nextButton = wrapper.find('[data-test="next-button"]')
         expect(nextButton.attributes().disabled).toEqual('disabled')
       })
 
-      it('is not disabled if hasNext is true', () => {
+      it('is enabled if hasNext is true', () => {
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-01-15T16:00:20Z" title="Wednesday, January 15th 2020, 5:00:20 pm +01:00">Jan 15, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -5,35 +5,31 @@ import Paginate from './Paginate'
 const localVue = global.localVue
 
 describe('Paginate.vue', () => {
-  let propsData, wrapper, nextButton, backButton
-
-  beforeEach(() => {
-    propsData = {}
-  })
+  let propsData = {}
+  let wrapper
+  let nextButton
+  let backButton
 
   const Wrapper = () => {
     return mount(Paginate, { propsData, localVue })
   }
-  describe('mount', () => {
-    beforeEach(() => {
-      wrapper = Wrapper()
-    })
 
+  describe('mount', () => {
     describe('next button', () => {
       beforeEach(() => {
         propsData.hasNext = true
         wrapper = Wrapper()
-        nextButton = wrapper.findAll('.ds-button').at(0)
+        nextButton = wrapper.find('[data-test="next-button"]')
       })
 
       it('is disabled by default', () => {
         propsData = {}
         wrapper = Wrapper()
-        nextButton = wrapper.findAll('.ds-button').at(0)
+        nextButton = wrapper.find('[data-test="next-button"]')
         expect(nextButton.attributes().disabled).toEqual('disabled')
       })
 
-      it('is not disabled if hasNext is true', () => {
+      it('is enabled if hasNext is true', () => {
```


better antonym for `disabled` :stuck_out_tongue: